app/Events/Website/WebsitePurged.php

```diff
@@ -27,11 +27,10 @@ class WebsitePurged
      * @param string $websiteJson the JSON string of the website
      * @param string $publicAdministrationJson the JSON string of the public administration
      */
-    public function __construct(string $websiteJson, string $publicAdministrationJson, ?string $userEmailForPublicAdministration = null)
+    public function __construct(string $websiteJson, string $publicAdministrationJson)
```

Comment:
Non riesco a vedere dove recuperi la mail da usare per l'invio di questa notifica.

Reply:
Nel caso di WebsitePurged le mail è inviata a tutti gli amministatori della PA di appartenenza del sito.
Effettivamente però l'attuale codice non prende in considerazione la mail specifica della PA ma quella presente in tabella users, quindi è da correggere.
Ho fatto una prima correzione nel commit 82b1bdb23b5d561108c62316c2d3964927e6116f, ma va fatta una verifica estesa a tutte le notifiche.